src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -24,6 +24,26 @@ s32 func_80A87DC0(Actor* thisx, PlayState* play);
 void func_80A87FD0(EnTru* this, PlayState* play);
 void func_80A881E0(EnTru* this, PlayState* play);
 
+typedef enum {
+    /* 0x00 */ KOUME_ANIM_INJURED_LYING_DOWN,
+    /* 0x01 */ KOUME_ANIM_INJURED_LYING_DOWN_MORPH,
+    /* 0x02 */ KOUME_ANIM_TRY_GET_UP,
+    /* 0x03 */ KOUME_ANIM_INJURED_RAISE_HEAD,
+    /* 0x04 */ KOUME_ANIM_INJURED_TALK,
+    /* 0x05 */ KOUME_ANIM_INJURED_HEAD_UP,
+    /* 0x06 */ KOUME_ANIM_INJURED_HEAD_UP_MORPH,
+    /* 0x07 */ KOUME_ANIM_TAKE,
+    /* 0x08 */ KOUME_ANIM_SHAKE, // Unused
+    /* 0x09 */ KOUME_ANIM_DRINK,
+    /* 0x0A */ KOUME_ANIM_FINISHED_DRINKING,
+    /* 0x0B */ KOUME_ANIM_HEALED,
+    /* 0x0C */ KOUME_ANIM_HOVER1,
+    /* 0x0D */ KOUME_ANIM_TAKE_OFF,
+    /* 0x0E */ KOUME_ANIM_FLY,
+    /* 0x0F */ KOUME_ANIM_HOVER2,
+    /* 0x10 */ KOUME_ANIM_MAX
+} KoumeAnimation;
```

Comment:
Since this seems to be the main actor of the two, this should probably go in the header and that header be included in both. There's obviously not much else shared since even the eyeIndex and blinkTimer are in different orders.

Reply:
Since the full animation array is duplicated, I think it makes more sense for the enum to be duplicated as well. (Though I should have changed the name/prefix to add an MT which I think I just forgot)

Reply:
Agreed. It would be better if we had a way to move both the enum and the array to a common place, but since we don't have any way to do that then this is fine.